api/errors/errors.go

```diff
@@ -1,47 +1,57 @@
 package errors
```

Comment:
This package name is already heavily overloaded. I'd recommend `errdefs`, as we've done in containerd.